scripts/move_to_spark_2.sh

```diff
@@ -2,10 +2,12 @@
 
 set +x
 
-svp="\${scala\.version\.prefix}"
+svp="\${scala.version.prefix}"
 substitution_cmd="s/_$svp/-spark2_$svp/g"
 
 find . -name "pom.xml" -exec sed \
+    -e "/utils-/ s/_2\.10/-spark2_2.10/g" \
+    -e "/adam-/ s/_2\.10/-spark2_2.10/g" \
```

Comment:
Does this mean we're missing `${scala.version.prefix}` somewhere, or are we intentionally not using it somewhere, in artifactId perhaps?